tests/Demo.py

```diff
@@ -227,37 +233,38 @@ def ParseCommandLine(config, args) :
     if options.private_key:
         private_key = options.private_key
     else:
-        #Generating the private Key for the client
+        # Generating the private Key for the client
         private_key = enclave_helper.generate_signing_keys()
 
     # Initializing Signature object, Worker Object
     sig_obj = signature.ClientSignature()
     worker_obj = worker.SGXWorkerDetails()
 
-    # -----------------------------------------------------------------
+
+# -----------------------------------------------------------------------------
 def Main(args=None):
     import config.config as pconfig
     import utility.logger as plogger
 
     # parse out the configuration file first
-    conffiles = [ "tcs_config.toml" ]
-    confpaths = [ ".", TCFHOME + "/config", "../../etc"]
+    conffiles = ["tcs_config.toml"]
+    confpaths = [".", TCFHOME + "/config", "../../etc"]
 
     parser = argparse.ArgumentParser()
-    parser.add_argument("--config", help="configuration file", nargs = "+")
-    parser.add_argument("--config-dir", help="configuration folder", nargs = "+")
+    parser.add_argument("--config", help="configuration file", nargs="+")
+    parser.add_argument("--config-dir", help="configuration folder", nargs="+")
```

Comment:
Is there any command to auto fix these lint issues which are reported by pylint?

Reply:
@Ram-srini A formatter for Python would be nice, similar to the Linux indent(1) program.
I don't know of any.